addon/services/options.js

```diff
@@ -49,5 +49,53 @@ export default Service.extend({
    */
   set(path, value) {
     this._data.set(path, value);
+  },
+
+  /**
+   * Registers a new component override.
+   *
+   * @param {Object} override The additional override.
+   * @param {String} override.label The text displayed in the form-builder dropdown.
+   * @param {String} override.component The path/name of the overriding component.
+   * @param {Array} [override.types] An optional question type restriction.
+   * @return {Void}
+   */
+  registerComponentOverride(override) {
+    const index = this._overrides.findIndex(
+      datum => datum.component === override.component
```

Comment:
datum?

Reply:
Oops. Forgot to update that one after switching from `_data` to `_overrides`. Should be `override`.

PS: datum (sing) -> data (pl)